client/src/features/LobbyPage/components/SelectCardsSet/SelectCardsSet.tsx

```diff
@@ -0,0 +1,65 @@
+import { FunctionComponent } from "react";
+import { InputRadio } from "../../../../components";
+import { SelectCardsProps } from "./types";
+import "./selectCardsSet.scss";
+
+export const SelectCardsSet: FunctionComponent<SelectCardsProps> = ({
+  onRegister,
+  errors,
+}): JSX.Element => {
+  return (
+    <div className="lobby-page__cards-value">
+      <h2>Choose cards deck</h2>
+      <InputRadio
```

Comment:
Prefer creating an array with necessary values and map over it returning InputRadio